assets/scss/components/_reminders.scss

```diff
@@ -0,0 +1,131 @@
+/*
+Reminder List
+
+Display an Unordered List of reminder items with time formatted in a new line.
+
+Markup:
+<ul class="reminder-list">
+  <li>Buy Envelope</li>
+  <li>Buy Stamp</li>
+  <li>Post form by <time>5:00PM</time></li>
+</ul>
+
+
+Styleguide Reminders.reminder-list
+*/
+
+
+.reminder-list {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  li {
+    padding: 0.5em 0;
+
+    time {
+      display: block;
+      color: $secondary-text-colour;
+
+      @include core-16;
+    }
+  }
+}
+
+
+/*
+Reminder
+
+Formats containing elements in a blue background and adds spacing.
+
+Markup:
+<div>
```

Comment:
no need for outer `<div>` remember this is an example of how to use this component so no superfluous markup please.